scripts/header

```diff
@@ -4,7 +4,7 @@ set -e
 cat <<END
 //  Ramda v$(node -p 'require("envvar").string("VERSION", require("./package.json").version)')
 //  https://github.com/ramda/ramda
-//  (c) 2013-$(git show -s --format=%ai | cut -d - -f 1) Scott Sauyet and Michael Hurley
+//  (c) 2013-$(git show -s --format=%ai | cut -d - -f 1) Scott Sauyet, Michael Hurley, and David Chambers
```

Comment:
I've always been a fan of the [Oxford comma][oc].

  [oc]: http://en.m.wikipedia.org/wiki/Serial_comma